service/local/splitFetcher.go

```diff
@@ -246,6 +246,28 @@ func (f *FileSplitFetcher) parseSplitsJson(data string) *dtos.SplitChangesDTO {
 	return &splitChangesDto
 }
 
+func (s *FileSplitFetcher) processSplitJson(data string, changeNumber int64) *dtos.SplitChangesDTO {
```

Comment:
you can return (*dtos.SplitChangesDTO, error) and in case only put `return s.processSplitJson`

Reply:
Updated.